protocol/messenger_communities.go

```diff
@@ -5878,11 +5888,7 @@ func (m *Messenger) processCommunityChanges(messageState *ReceivedMessageState)
 			}
 
 		} else if changes.ShouldMemberLeave {
```

Comment:
Would be great to rename it as well to make the intention a bit clearer.
```suggestion
		} else if changes.Kicked {
```